packages/wp-now/src/run-cli.ts

```diff
@@ -35,22 +36,22 @@ export async function runCli() {
 					describe:
 						'Path to the PHP or WordPress project. Defaults to the current working directory.',
 					type: 'string',
-					default: process.cwd(),
 				});
 				yargs.option('php', {
 					describe: 'PHP version to use.',
 					type: 'string',
-					default: DEFAULT_PHP_VERSION,
 				});
 			},
 			async (argv) => {
 				const spinner = startSpinner('Starting the server...');
 				try {
+					const options = await ConfigFactory.getInstance().read({
```

Comment:
I wonder if ConfigFactory is needed at all here, couldn't this say just:

```suggestion
					const options = {
							projectPath: argv.path as string,
							phpVersion: argv.php as SupportedPHPVersion
					};
```

Reply:
The ConfigFactory is responsible to merge all the configs, both passed arguments and wp-env, if any.
On top of that, it applies the defaults if none of those props exist.
This is why I use it here.
Why do you think it's not needed here?

Reply:
Gotcha, I got confused :-) It would be nice to just call a function such as `getWpNowConfig()` or so